cip-governance-metadata-extension/README.md

```diff
@@ -0,0 +1,228 @@
+Created: 2025-10-15
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP extends [CIP-100 | Governance Metadata][CIP-100] to introduce a standardized `onChain` property.
+The `onChain` property encapsulates the on-chain effects using [CIP-116 | Standard JSON encoding for Domain Types][CIP-116] standard encoding,
+enabling verification that metadata content matches on-chain action and preventing metadata replay attacks.
+This can be applied to all types of governance metadata, including governance actions, votes, DRep registrations/updates, and Constitutional Committee resignations.
+
+## Motivation: why is this CIP necessary?
+
+Without a standardized mechanism to bind metadata to specific on-chain effects,
+there is a security vulnerability in **metadata replay attacks**.
+
+This vulnerability has been discussed via [CIPs Issue #970](https://github.com/cardano-foundation/CIPs/issues/970) by [Crypto2099](https://github.com/Crypto2099) and further within [CIPs Issue #978](http://github.com/cardano-foundation/CIPs/issues/978) by [gitmachtl](https://github.com/gitmachtl).
+
+### Metadata Replay
+
+A malicious actor could;
+
+1. Copy the metadata from a legitimate treasury withdrawal governance action
+2. Modify only the destination address in the on-chain action
+3. Submit this as a new governance action with the copied metadata
+
+From a voter's perspective, examining the metadata would show legitimate content (proper title, abstract, rationale, references, and author signatures).
+However, the actual on-chain effect would send funds to a different, malicious address.
+
+In high-volume voting environments such attacks could succeed if voters don't manually verify that the metadata matches the on-chain effects.
+Relying on manual verification by voters is error prone and inefficient.
+
+### Other Author attacks
+
+- discuss attacks where multiple authors are collaborating before constructing on-chain part
+- having an onChain property prevents someone from attaching the signed metadata to the wrong thing
+
+## Specification
+
+### The `onChain` Property
+
+The `onChain` property is a new **optional** property within the `body` object of [CIP-100][] governance metadata.
+
+#### JSON-LD Context
+
+The `onChain` property shall be defined in the JSON-LD `@context` as follows:
+
+```json
+{
+  "@context": {
+    "CIP116": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0116/README.md#",
+    "body": {
+      "@id": "CIP100:body",
+      "@context": {
+        "onChain": {
+          "@id": "CIPXXX:onChain"
+        }
+      }
+    }
+  }
+}
+```
+
+> Note: `CIPXXX` represents this CIP's number once assigned.
+
+#### Structure
+
+The `onChain` property **MUST** conform to one of the [CIP-116][] Conway-era governance types.
+The specific type is indicated by the `@type` field within the `onChain` object.
+
+> ⚠️ Note: any entry of anchor properties within the CIP116 objects must be removed as they constitute a circular dependency
```

Comment:
```suggestion
> [!WARNING]
> Any entry of anchor properties within the CIP116 objects must be removed as they constitute a circular dependency
```
> [!WARNING]
> When that "note" is really a "warning" (as per the symbol) you can use this syntax to include a formatted Warning.

The 5 possible "alerts" are described here: [GitHub Docs > Basic writing and formatting syntax > Alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)